.circleci/config.yml

```diff
@@ -55,6 +55,126 @@ disable_telemetry: &disable_telemetry
   run:
     name: Disable Telemetry
     command: vendor/bin/blt blt:telemetry:disable --no-interaction
+#Jobs
```

Comment:
@hectorlj SWS just removed this in a [recent PR](https://github.com/SU-HSDO/suhumsci/commit/09f8725349e2e7c596132bf738b9966d19974efe) to use Github Actions instead of CircleCI. Are you bringing it back for a specific reason or is this just a mistake?

Reply:
Good catch! Looks like this was a mistake that happened when merging develop into this branch. Not surprising considering how old the branch is. I'll remove it